spinalcordtoolbox/testing/create_test_data.py

```diff
@@ -132,12 +133,29 @@ def dummy_segmentation(size_arr=(256, 256, 256), pixdim=(1, 1, 1), dtype=np.floa
     nx, ny, nz = [int(size_arr[i] * pixdim[i]) for i in range(3)]
     data = np.random.random((nx, ny, nz)) * 0.
     xx, yy = np.mgrid[:nx, :ny]
-    # loop across slices and add object
-    for iz in range(nz):
-        if shape == 'rectangle':  # theoretical CSA: (a*2+1)(b*2+1)
-            data[:, :, iz] = ((abs(xx - nx / 2) <= radius_RL) & (abs(yy - ny / 2) <= radius_AP)) * 1
-        if shape == 'ellipse':
-            data[:, :, iz] = (((xx - nx / 2) / radius_RL) ** 2 + ((yy - ny / 2) / radius_AP) ** 2 <= 1) * 1
+    if not interleaved:
+        # loop across slices and add object
+        for iz in range(nz):
+            if shape == 'rectangle':  # theoretical CSA: (a*2+1)(b*2+1)
+                data[:, :, iz] = ((abs(xx - nx / 2) <= radius_RL) & (abs(yy - ny / 2) <= radius_AP)) * 1
+            if shape == 'ellipse':
+                data[:, :, iz] = (((xx - nx / 2) / radius_RL) ** 2 + ((yy - ny / 2) / radius_AP) ** 2 <= 1) * 1
+    elif interleaved:
+        import numpy.matlib
```

Comment:
all imports should go at the top of the file according to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)